### PR TITLE
chore: ECE phpunit test location fix

### PR DESCRIPTION
**Pull request by frosso**
Fixes #

#### Changes proposed in this Pull Request

I noticed that the ECE unit test location didn't match the location of the classes they were testing against.
The classes are in an `express-checkout` directory, but the unit tests aren't.
Fixing.

#### Testing instructions

No change in functionality - this is just tests being moved around.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

File: tests/unit/express-checkout/test-class-wc-payments-express-checkout-button-display-handler.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 use WCPay\WooPay\WooPay_Utilities;
 
 /**
- * WC_Payments_WooPay_Button_Handler_Test class.
```
Comment (frosso) [inline tests/unit/express-checkout/test-class-wc-payments-express-checkout-button-display-handler.php:16]: Fixed the class' description to reflect the class in the file.

File: tests/unit/express-checkout/test-class-wc-payments-express-checkout-button-helper.php [hunk 2]
```diff
@@ -11,7 +11,7 @@
 use WCPay\Session_Rate_Limiter;
 
 /**
- * WC_Payments_Payment_Request_Button_Handler_Test class.
```
Comment (frosso) [inline tests/unit/express-checkout/test-class-wc-payments-express-checkout-button-helper.php:14]: Same as above.

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `9758` or branch name `chore/ece-php-tests-location` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=chore/ece-php-tests-location) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 043fe49183f7e77bf69a50d5b54bec66fd4ca4ab
- Build time: 2024-11-19 16:35:13 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: tests/unit/express-checkout/test-class-wc-payments-express-checkout-button-display-handler.php (hunk 1 above)
Comment (FangedParakeet) [reply tests/unit/express-checkout/test-class-wc-payments-express-checkout-button-display-handler.php:16]: 💾 + 🧴. 🤦

**Review by FangedParakeet — APPROVED** [review]
Cool, LGTM. 🚀

**Pull request merged**